i18n/en-US.yml

```diff
@@ -0,0 +1,30 @@
+_id: en-US
+_name: English
+components:
+  DefaultItinerary:
```

Comment:
I've never been a fan of organizing messages by components. Often times messages can come from util functions shared across multiple components. I recommend organizing messages by overall category and avoiding deep hierarchical organizations if at all possible.